src/utils.js

```diff
@@ -15,9 +15,14 @@ export function getOptions(pluginOptions) {
 }
 
 export function absoluteUrlForLocation(location) {
-  return `${window.location.protocol}//${window.location.host}${
-    location.pathname
-    }${location.search}${location.hash}`;
+  return [
+    window.location.protocol,
+    "//",
+    window.location.host,
+    location.pathname,
+    location.search,
+    location.hash,
+  ].join("")
```

Comment:
This change is completely fine ofc but plz try not to slip these kinds of things into PRs about separate issues.